src/Umbraco.Web.UI/umbraco/config/lang/da.xml

```diff
@@ -1004,17 +1005,91 @@ Mange hilsner fra Umbraco robotten
-    <key alias="insertDictionaryItem">Indsæt ordbogselement</key>
-    <key alias="insertMacro">Indsæt makro</key>
-    <key alias="insertPageField">Indsæt Umbraco sidefelt</key>
+    <key alias="insertContentAreaPlaceHolder">Indsæt indholdsområde placeholder</key>
+
+    <key alias="insert">Indsæt</key>
+    <key alias="insertDesc">Hvad vil du indsætte ?</key>
+
+    <key alias="insertDictionaryItem">Oversættelse</key>
+    <key alias="insertDictionaryItemDesc">Indsætter en oversætbar tekst som skifter efter det sprog som websitet vises i.</key>
+
+    <key alias="insertMacro">Makro</key>
+    <key alias="insertMacroDesc">
+      En makro er et element som kan have forskellige indstillinger når det indsættes.
+      Brug det som en genbrugelig del af dit design såsom gallerier, formularer
+      og lister.
+    </key>
+
+    <key alias="insertPageField">Side værdi</key>
+    <key alias="insertPageFieldDesc">
+      Viser værdien af et felt fra den nuværende side. Kan indstilles til at bruge rekursive værdier eller
+      vise en standard værdi i tilfælde af at feltet er tomt.
+    </key>
+
+    <key alias="insertPartialView">Partial view</key>
+    <key alias="insertPartialViewDesc">
+      Et Partial View er et skabelon element som kan indsættes i andre skabeloner og derved
+      genbruges og deles på tværs af side-skabelonerne.
+    </key>
+
     <key alias="mastertemplate">Master skabelon</key>
-    <key alias="quickGuide">Lynguide til Umbracos skabelontags</key>
+    <key alias="noMastertemplate">Ingen master skabelon</key>
+    <key alias="noMaster">Ingen master</key>
+
+    <key alias="renderBody">Indsæt en underliggende skabelon</key>
+    <key alias="renderBodyDesc">
+      <![CDATA[
+      Henter indholdet af en underliggende skabelon ind, ved at 
+      indsætte et <code>@RenderBody()</code> element.
+      ]]>
+    </key>
+
+
+    <key alias="defineSection">Definer en sektion</key>
+    <key alias="defineSectionDesc">
+      <![CDATA[
+         Definerer en del af din skabelon som en navngivet sektion, ved at 
+         omkranse den i <code>@section { ... }</code>. Herefter kan denne sektion flettes ind i 
+         overliggende skabelon ved at indsætte et <code>@RenderSection</code> element.
+      ]]>
+    </key>
+
+    <key alias="renderSection">Indsæt en sektion</key>
+    <key alias="renderSectionDesc">
+      <![CDATA[
+      Henter indholdet af en sektion fra den underliggende skabelon ind, ved at indsætte et 
+      <code>@RenderSection(name)</code> element. Den underliggende skabelon skal have 
+      defineret en sektion via et <code>@section [name]{ ... }</code> element.
+      ]]>
+    </key>
+
+    <key alias="sectionName">Sektion navn</key>
+    <key alias="sectionMandatory">Sektionen er oblikatorisk</key>
+    
+    <key alias="sectionMandatoryDesc">
+      Hvis oblikatorisk, skal under-skabelonen indeholde en <code>@section</code> definition. 

```

Comment:
Hvis **obligatorisk**, skal **underskabelonen** indeholde en <code>@section</code>-definition. 
Underskabelonen in one word, fixed spelling mistake and added a hyphen before definition